README.md

```diff
@@ -84,7 +84,7 @@ gem install ruboclean
 ## Command synopsis
 
 ```shell
-ruboclean [path] [--silent] [--preserve-comments] [--preserve-paths]
+ruboclean [path] [--silent] [--preserve-comments] [--preserve-paths] [--verify]
```

Comment:
I feel like the flag's name could be improved. What about 

* `--verify-only` or even
* `--dry-run`

What's your opinion?

Reply:
I feel `--verify` represents a behavior of this flag the best. `--verify-only` sounds like it also verifies, when this flag is not passed. `--dry-run` doesn't sound like it exit with 1.

Also, other ruby tool `tapioca` also uses `--verify` for the same future. 
https://github.com/Shopify/tapioca?tab=readme-ov-file#keeping-rbi-files-for-dsls-up-to-date

Reply:
I feel `--verify` represents a behavior of this flag the best. `--verify-only` sounds like it also verifies, when this flag is not passed. `--dry-run` doesn't sound like it exit with `1`.

Also, other ruby tool tapioca also uses `--verify` for the same future.
https://github.com/Shopify/tapioca?tab=readme-ov-file#keeping-rbi-files-for-dsls-up-to-date

Reply:
Good argumentation, I can see now. Thank you!